regen/feature.pl

```diff
@@ -943,7 +942,16 @@ =head2 The 'extra_paired_delimiters' feature
 will want to ensure the parser will decode the source code from UTF-8 bytes
 with a declaration such as C<use utf8>.
 
-This feature is available starting in Perl 5.36.
+This feature is available starting in Perl 5.36.  From Perl 5.36 to 5.38,
+it was classed as experimental, and Perl emitted a warning for its usage,
+except when explicitly disabled:
+
+    no warnings "experimental::extra_paired_delimiters";
+
+As of Perl 5.40, use of this feature no longer triggers a warning (though the
+C<experimental::extra_paired_delimiters> warning category stilll exists for
```

Comment:
```suggestion
C<experimental::extra_paired_delimiters> warning category still exists for
```

Reply:
Ahyes. Annoyingly, I copied that typo from the `isa` section. I'll fix that up separately.